packages/TextField/TextField.vue

```diff
@@ -1,29 +1,47 @@
 <template>
   <label>
     {{label}}
-    <input
-      :type="type"
-      :placeholder="placeholder"
-      :value="value"
-      @input="$emit(`input`, $event.target.value)"
-    />
+    <div class="password">
```

Comment:
You should put another class name. When the type is text it doesn't make sense to have a password class

Reply:
Fixed.